importer/tag/src/main/java/org/itsallcode/openfasttrace/importer/tag/LongTagImportingLineConsumer.java

```diff
@@ -28,7 +28,10 @@ class LongTagImportingLineConsumer extends RegexLineConsumer
             + "," + OPTIONAL_WHITESPACE + "\\p{Alpha}+)*)";
     private static final String TAG_REGEX = TAG_PREFIX + OPTIONAL_WHITESPACE//
             + "(" + COVERING_ARTIFACT_TYPE_PATTERN + ")"
-            + optional(SpecificationItemId.ARTIFACT_TYPE_SEPARATOR + SpecificationItemId.REVISION_SEPARATOR
+            + optional(SpecificationItemId.ARTIFACT_TYPE_SEPARATOR
```

Comment:
At least for me the `optional()` wrapper method does not make the RegEx more readable.